### PR TITLE
chore: rename monday-ui-react-core to @vibe/core

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/7217234090

### CODE REVIEW

File: packages/core/src/components/Icon/__stories__/Icon.mdx [hunk 1]
```diff
@@ -63,10 +63,10 @@ As a default the icon will inherit the color of it's parent container
 
 ## Icons List
 
-Icons are exported by name from `monday-ui-react-core/icons`:
+Icons are exported by name from `@vibe/core/icons`:
```
Comment (YossiSaadi) [inline packages/core/src/components/Icon/__stories__/Icon.mdx:66]: @vibe/icons?

File: packages/core/src/components/Icon/__stories__/Icon.mdx [hunk 2]
```diff
@@ -63,10 +63,10 @@ As a default the icon will inherit the color of it's parent container
...
 
 ```javascript
-import { DoubleCheck } from "monday-ui-react-core/icons";
+import { DoubleCheck } from "@vibe/core/icons";
```
Comment (YossiSaadi) [inline packages/core/src/components/Icon/__stories__/Icon.mdx:69]: @vibe/icons?

File: packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.mdx [hunk 3]
```diff
@@ -19,7 +19,7 @@ import * as UseActiveDescendantListFocusStories from "./useActiveDescendantListF
 ### Import
 
 ```js
-import { useActiveDescendantListFocus } from "monday-ui-react-core/hooks";
+import { useActiveDescendantListFocus } from "@vibe/core/hooks";
```
Comment (YossiSaadi) [inline packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.mdx:22]: do we want to have it under @vibe/hooks?

File: packages/core/src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.tsx [hunk 4]
```diff
@@ -60,6 +60,7 @@ export const OtherContributorsList = () => {
       </Paragraph>
       <GithubContributorsList
         organizationName="mondaycom"
+        // TODO: need to support more than 1 package
```
Comment (YossiSaadi) [inline packages/core/src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.tsx:63]: wdym?

File: packages/storybook-blocks/jest.config.ts [hunk 5]
```diff
@@ -14,7 +14,7 @@ module.exports = {
   preset: 'ts-jest/presets/default',
   testEnvironment: 'jest-environment-jsdom',
   testMatch: ['**/__tests__/**/*.test.[jt]s?(x)'],
-  transformIgnorePatterns: ['/node_modules/(?!monday-ui-react-core)'],
+  transformIgnorePatterns: ['/node_modules/(?!@vibe/core)'],
```
Comment (YossiSaadi) [inline packages/storybook-blocks/jest.config.ts:17]: it sound like a mistake, why is it inside storybook-blocks?

**Review by YossiSaadi — APPROVED** [review]

File: packages/core/src/components/Icon/__stories__/Icon.mdx (hunk 1 above)
Comment (talkor) [reply packages/core/src/components/Icon/__stories__/Icon.mdx:66]: Good catches! fixing

File: packages/core/src/components/Icon/__stories__/Icon.mdx (hunk 2 above)
Comment (talkor) [reply packages/core/src/components/Icon/__stories__/Icon.mdx:69]: Good thing you DoubleCheck

File: packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.mdx (hunk 3 above)
Comment (talkor) [reply packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.mdx:22]: I'm all for it but it's not a blocker for the release

File: packages/core/src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.tsx (hunk 4 above)
Comment (talkor) [reply packages/core/src/storybook/stand-alone-documentaion/welcome/other-contributors-list/other-contributors-list.tsx:63]: Actually it's per repo, not npm package, changing to "vibe"

File: packages/storybook-blocks/jest.config.ts (hunk 5 above)
Comment (talkor) [reply packages/storybook-blocks/jest.config.ts:17]: You're right, removing

**Review by YossiSaadi — APPROVED** [review]

**Pull request merged**